source/Octopus.Server.Client/Model/Endpoints/PollingKubernetesTentacleEndpointConfigurationResource.cs

```diff
@@ -0,0 +1,15 @@
+namespace Octopus.Client.Model.Endpoints
+{
+    public class PollingKubernetesTentacleEndpointConfigurationResource : KubernetesTentacleEndpointConfigurationResource, IPollingTentacleEndpointResource
```

Comment:
This is a rename for `PollingTentacleEndpointConfigurationReosurce` to PollingKubernetesTentacleEndpointConfigurationResource`.

Reply:
Reverted as not worth it.